blog/rome-incentives/README.md

```diff
@@ -0,0 +1,225 @@
+# Post
+
+#### Title
+
+The Rome Incentive Structure
+
+#### Purpose
+
+Inform our community about Rome's incentive structure.
+
+#### url
+
+blog.joystream.org/rome-incentives/
+
+#### Cover
+
+<p align="center"><img src="rome-incentives.png"></p>
+
+#### Lead
+
+Rome offers a new incentive structure for our testnet participants.
+
+#### Body
+
+As with Sparta, Athens and Acropolis we plan to pay users in monero (XMR) to participate on our upcoming testnet release, codenamed Rome. Our fifth testnet ambitiously introduces two new paid roles which we are extremely excited about!
```

Comment:
- Perhaps add a link to previous post, and announce the release date for this testnet
- Add that guides will be up Sunday
- Finally, add that unlike previous testnets, this release will have *dynamic* incentives, in order to
   - Allocate resources
   - Ensure good uptime/response/etc.
   - Prepare for next testnet, that will shift the responsibility of setting rewards to the council

Reply:
(Doesn't have to be a bullet point list)